Index: countnames.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/countnames.c b/countnames.c
--- a/countnames.c	
+++ b/countnames.c	(date 1676852548980)
@@ -37,7 +37,7 @@
             int i = 0;
             while(i < NUM_NAME) {
                 if (i >= numUniqueName) {
-                    names[i] = temp;
+                    strcpy(names[i],temp);
                     numUniqueName++;
                     count[i]++;
                     i = NUM_NAME;
